Instruments/source/BTagEffEstimator.cxx

```diff
@@ -9,8 +9,8 @@ This file is part of https://github.com/hh-italian-group/hh-bbtautau. */
 #include "h-tautau/Analysis/include/EventTuple.h"
 #include "AnalysisTools/Core/include/AnalyzerData.h"
 #include "h-tautau/Analysis/include/AnalysisTypes.h"
-#include "h-tautau/Cuts/include/Btag_2016.h"
-#include "h-tautau/Cuts/include/hh_bbtautau_2016.h"
```

Comment:
Please, keep nominal support for 2016. For example, pass period as a command line argument.